tests/jest_integration/jest_project/integration.fuzz.js

```diff
@@ -37,6 +37,14 @@ describe("Jest Integration", () => {
 		target.fuzzMe(data);
 	});
 
+	it.fuzz(
+		"execute sync hashed fuzz test with dictionary",
+		(data) => {
+			target.fuzzMeHashed(data);
+		},
+		{ dictionaries: ["Amazing"] },
```

Comment:
Can you add/extend a test with multiple dictionary entries? Maybe make it twice amazing :smile: